public/openapi3.json

```diff
@@ -8967,7 +8979,7 @@
       "QueryDataResponse": {
         "description": "It is the return type of a QueryData call.",
         "properties": {
-          "Responses": {
+          "results": {
```

Comment:
interesting -- this is based on the changes in https://github.com/grafana/grafana/pull/84010

BUT the swagger here is kinda silly/wrong anyway -- the types from sdk (data.Frame etc) do not serialize in a form matching the default go marshal behavior